Pipfile

```diff
@@ -0,0 +1,26 @@
+[[source]]
+name = "pypi"
+url = "https://pypi.org/simple"
+verify_ssl = true
+
+[dev-packages]
+
+[packages]
+ipython = "*"
+sphinx-autobuild = "*"
+sphinxcontrib-plantuml = "*"
+recommonmark = "*"
+sphinx-rtd-theme = "*"
+pytest-bdd = "<4.0.0"
+pytest-cov = "*"
+pytest-mock = "*"
+pytest-catchlog = "*"
+pytest-subtests = "*"
+pytest-twisted = "*"
+coverage = "*"
+pytest-runner = "*"
+pytest = "*"
+mock = "*"
+ebu-tt-live = {editable = true,file = "file:///Users/megitn02/Code/bbc/ebu-tt-live-toolkit"}
```

Comment:
is this meant to be the branch? `https://github.com/bbc/ebu-tt-live-toolkit/MSPA-1576-br-children-of-untimed-p`

Reply:
Er, hmm, yes, it definitely shouldn't be a local file path!

Reply:
I don't know why the file is there at all, it doesn't seem to be needed.

Reply:
File removed, so this problem should have gone away!